contracts/third_party/curve/Gauge.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity 0.8.4;
+
+interface Gauge {
+    function deposit(uint) external;
+    function balanceOf(address) external view returns (uint);
+    function claimable_reward(address _addr, address _token) external view returns (uint256);
+    function claimed_reward(address _addr, address _token) external view returns (uint256);
+    function claimable_reward_write(address _addr, address _token) external returns (uint256);
+    function withdraw(uint, bool) external;
```

Comment:
add attributes names, and better with comment from code
def withdraw(_value: uint256, _claim_rewards: bool = False):
    """
    @notice Withdraw `_value` LP tokens
    @dev Withdrawing also claims pending reward tokens
    @param _value Number of tokens to withdraw
    """